src/resources/api/auth.js

```diff
@@ -1,23 +1,16 @@
 import api from "./config";
 
-let apiPrefix = "/badaso-api";
+let apiPrefix = "/badaso-api/module/lms/v1/auth/";
 
 export default {
   register(data) {
-    let ep = apiPrefix + "/module/lms/v1/auth/register";
+    let ep = apiPrefix + "register";
```

Comment:
woops didn't notice this one before. `ep` is not clear enough IMO, please change it to something like `url`